src/components/NewExperiment.vue

```diff
@@ -1699,14 +1702,40 @@ export default Vue.extend({
       }
 
       dialogSelection.then(rowPairsFromDialog => {
-        // parsedRows = [[["name", "a"], ["measurement", 5], ["uncertainty", null]]]
-        parsedRows.forEach((rowPairs, rowIx) => {
-          if (!table.items[rowOffset + rowIx]) {
-            // Create excess rows.
-            table.items.push({ temporaryId: uuidv4() });
-          }
-          [...rowPairs, ...rowPairsFromDialog].forEach(([property, value]) => {
-            Vue.set(table.items[rowOffset + rowIx], property, value);
+        // Transpose the array, so we can make batch requests
+        const columns = zip(...rows);
+        // TODO: handle excess columns
+        const parsedColumnsPromises = columns.map((column, columnIx) => {
+          const property = table.headers[columnOffset + columnIx].value;
+          const parsedColumnPromise = table.parsePasted[property](column, {
```

Comment:
Oh, we can't have `if (cell) { parse } else { null }` now that multiple cells (=column) is parsed. We don't handle pasting empty cells well anymore.

And I'm not a fan of delegating it into every parsePasted function
```diff
-strs => strs.map(str => parseFloat(str))
+strs => strs.map(str => str ? parseFloat(str) : null)
```

TODO
- [ ] think